backend/services/ai/remediation/router.py

```
@@ -0,0 +1,307 @@
+    if finding_type == "missing_mfa":
+        return _remediate_missing_mfa(input_, risk_level)
+    if finding_type == "old_access_key":
+        return _remediate_old_access_key(input_, risk_level)
+    if finding_type == "unused_access_key":
+        return _remediate_unused_access_key(input_, risk_level)
+
+    # Out of scope (guardrails will handle blocking later)
+    return make_response(
+        fix_type="out_of_scope",
+        risk_level="low",
+        explanation=f"Unsupported finding_type: {input_.finding_type}. Only the project’s top 10 IAM finding types are handled in this sprint.",
```

Ruff (0.15.7)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)